examples/jsm/geometries/TextGeometry.js

```diff
@@ -53,12 +54,39 @@ class TextGeometry extends ExtrudeGeometry {
 
 			super( shapes, parameters );
 
+			// for conversion of font to object units (ie. px -> m)
+
+			const scale = parameters.scale;
+
+			if ( scale !== undefined ) {
+
+				this.computeBoundingBox();
```

Comment:
Why is the bounding box computed at this point? It will be automatically recomputed when calling `BufferGeometry.scale()`?

Reply:
BTW: Is the introduction of a `scale` parameter really necessary? Couldn't apps just use `Object3D.scale` instead?

Reply:
the reason to do it here is getting a scaled down (0.01) geometry mostly for connivence to be used in Editor.

the `object.scale` could be used, but I wasn't sure if it was a good idea to insert a TextGeometry at a custom object scale, hence having it in the geometry seems like a more convenient way.

Reply:
In order to keep the parameters of `TextGeometry` to a minimum, would be better to use `Object3D.scale` and not introduce an additional way for transforming geometry.